specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json

```diff
@@ -0,0 +1,575 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-12-01",
+    "title": "WebApplicationFirewallManagement",
+    "description": "APIs to manage web application firewall rules"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies": {
+      "get": {
+        "tags": [
+          "ListWebApplicationFirewallPolicies"
+        ],
+        "description": "Lists all of the protection policies within a resource group.",
+        "operationId": "Policies_List",
```

Comment:
`Policies` seems too generic. Could you please rename all `operationId` from `Policies` to `WebApplicationFirewallPolicies`?